collect_app/src/main/java/org/odk/collect/android/receivers/NetworkReceiver.java

```diff
@@ -176,6 +176,22 @@ private void uploadForms(Context context) {
         }
     }
 
+    private boolean submitInstance(String jrFormId, boolean autoSendSettings) {
```

Comment:
I would call this something like `isFormAutoSendEnabled` to clarify what it does. I think that will make the code that uses it easier to read.